addons/point_of_sale/static/src/app/generic_components/numpad/numpad.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates id="template" xml:space="preserve">
   <t t-name="point_of_sale.Numpad">
-    <div t-attf-class="{{props.class}} numpad row row-cols-{{props.buttons.length / 4}} gx-0">
+    <div t-attf-class="{{props.class}} numpad row row-cols-{{ props.options.columns }} gx-0">
```

Comment:
you pass the option `columns: 3` but the old code would have given the same result

Reply:
I know, but it wasn't clear before, and what would happen if we didn't have a number that was an even number?

Reply:
Okay, I've added a docstring that explains the 12 or 16 button limitation.